src/main/java/com/amazonaws/services/kinesis/multilang/MultiLangDaemonConfig.java

```diff
@@ -100,6 +117,29 @@ public MultiLangDaemonConfig(String propertiesFile,
         prepare(processingLanguage);
     }
 
+    private ClientConfiguration buildClientConfig(final String serviceName, KinesisClientLibConfiguration config)
+    {
+        try
+        {
+            Region region = RegionUtils.getRegion(config.getRegionName());
+            String serviceEndpoint = serviceName + "." + region.getName() + "." + region.getDomain();
```

Comment:
FYI - I know this should be using `Region#getServiceEndpoint()`, but the [regions.xml](https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/resources/com/amazonaws/regions/regions.xml) that ships with even the latest AWS SDK doesn't include us-west-1 as a valid region for Kinesis endpoint ...